folium/features.py

```diff
@@ -809,95 +809,81 @@ class Circle(Marker):
     location: tuple or list, default None
         Latitude and Longitude of Marker (Northing, Easting)
     radius: int
-        The radius of the circle in meters. For setting the radius in pixel,
-        use CircleMarker.
-    color: str, default 'black'
+        The radius of the circle in meters.
+        For setting the radius in pixel, use CircleMarker.
+    color: str, default '#3388ff'
         The color of the marker's edge in a HTML-compatible format.
-    fill_color: str, default 'black'
+    fill: bool, default False
+        If true the circle will be filled.
+    fill_color: str, default to the same as color
         The fill color of the marker in a HTML-compatible format.
-    fill_opacity: float, default 0.6
+    fill_opacity: float, default 0.2
         The fill opacity of the marker, between 0. and 1.
     popup: string or folium.Popup, default None
         Input text or visualization for object.
 
+    See http://leafletjs.com/reference-1.2.0.html#path for more otions.
+
     """
-    def __init__(self, location, radius=500, color='black',
-                 fill_color='black', fill_opacity=0.6, popup=None):
-        super(Circle, self).__init__(
-            _validate_location(location),
-            popup=popup
-        )
-        self._name = 'Circle'
-        self.radius = radius
-        self.color = color
-        self.fill_color = fill_color
-        self.fill_opacity = fill_opacity
+    def __init__(self, location, radius=10, popup=None, **kw):
+        super(Circle, self).__init__(_validate_location(location), popup=popup)
+        self._name = 'circle'
+        options = _parse_path(**kw)
+
+        options.update({'radius': radius})
+        self.options = json.dumps(options, sort_keys=True, indent=2)
 
         self._template = Template(u"""
             {% macro script(this, kwargs) %}
 
             var {{this.get_name()}} = L.circle(
                 [{{this.location[0]}},{{this.location[1]}}],
-                {{ this.radius }},
-                {
-                    color: '{{ this.color }}',
-                    fillColor: '{{ this.fill_color }}',
-                    fillOpacity: {{ this.fill_opacity }}
-                    }
-                )
-                .addTo({{this._parent.get_name()}});
+                {{ this.options }}
```

Comment:
We should use `json.dumps` everywhere to reduce the amount of code like `'true' if something else 'false'`.